### ISSUE TITLE
strip "v" prefix from phys2bids.__version__, make it semver

### ISSUE THREAD

**Issue by yarikoptic** [physiopy/phys2bids#311]
ATM
```
$> python -c 'import phys2bids; print(phys2bids.__version__)'
v2.1.0+119.gccd39e8
$> git co v2.1.0
$> python -c 'import phys2bids; print(phys2bids.__version__)'
v2.1.0
```
which is awesome that it includes git information, but 

- not good that has `v` prefix.  `v` in git tags is ok, but undesired for `__version__` which should be just the version .  Check other modules (e.g. numpy etc) -- they all lack any `v` prefix in `__version__`.  Dropping it would make `__version__` immediately a proper semver (although "build" doesn't support sorting, but sorting of git states even with index should not be relied upon anyways): 

```
In [1]: import semver                                                                                                   

In [2]: semver.parse("v2.1.0+119.gccd39e8")                                                                             
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-8d6227dd9bc8> in <module>
----> 1 semver.parse("v2.1.0+119.gccd39e8")

/usr/lib/python3/dist-packages/semver.py in parse(version)
     19     match = _REGEX.match(version)
     20     if match is None:
---> 21         raise ValueError('%s is not valid SemVer string' % version)
     22 
     23     verinfo = match.groupdict()

ValueError: v2.1.0+119.gccd39e8 is not valid SemVer string

In [3]: semver.parse("2.1.0+119.gccd39e8")                                                                              
Out[3]: 
{'major': 2,
 'minor': 1,
 'patch': 0,
 'prerelease': None,
 'build': '119.gccd39e8'}
```

related:  please use annotated git tags (although that would require tagging locally and then minting a release from pushed tag on github).  I typically tag locally, add changelog entry into its annotation, push to github , go to releases, edit release for the tag and do not enter/change anything, just "save" it and then it reuses annotation from the git tag (changelog).   

```shell
(git)lena:~/proj/misc/phys2bids[tags/v2.1.0]git
$> git describe       
fatal: No annotated tags can describe '4e0cf9c8683d11528e01c8e3d7d63e179c6bd6f2'.
However, there were unannotated tags: try --tags.
$> git describe --tags
v2.1.0
```

**Comment by smoia** [comment]
@yarikoptic thank you for your suggestion!
We're actually trying to move toward an automatic tag/release pipeline. Do you think (or know if) there is a way to still have annotated tags with automatic release? @hipstersmoothie is `auto` doing this already, or would it be possible to implement this feature?

**Comment by hipstersmoothie** [comment]
1. The tags `auto` creates are annotated
2. There is a config option for this. it was missing from the docs https://github.com/intuit/auto/pull/1577/files

**Comment by smoia** [comment]
Oh, that's great news! So we really need to finish implementing auto for this to be solved - @yarikoptic do you think it's necessary to strip the prefix from existing versions too? I would much like to avoid it!

**Comment by yarikoptic** [comment]
I am "forward looking" -- otherwise we would need to port all bug fixes to all previous releases of everything and have no time for actually going forward ;)

**Comment by smoia** [comment]
:rocket: Issue was released in [`2.2.1`](https://github.com/physiopy/phys2bids/releases/tag/2.2.1) :rocket:
